backends/samsung/README.md

```diff
@@ -0,0 +1,71 @@
+# ExecuTorch Samsung Exynos Delegate
+
+The subtree contains Exynos delegation implementation for ExecuTorch. The target of delegation
+is deploying torch model run with exynos NPU/DSP.
+
+This backend is implemented on the top of [EXYNOS_LITECORE](https://soc-developer.semiconductor.samsung.com/global/development/light-core)
+Please prepare the SDK before you start, it is important to code compilation and runtime.
+
+## Delegate Options
+
+### Supported Chipset
+- Exynos 2500 (E9955)
+
+### Supported Inference Type
+- Quantized (i8/u8/i16/u16)
```

Comment:
what about int4? is it under dev?

Reply:
Created an issue to track: https://github.com/pytorch/executorch/issues/14003